src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -108,9 +110,29 @@ internal abstract class BrokerClient : IBrokerClient {
         public virtual async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, string rCommandLineArguments = null, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
             DisposableBag.ThrowIfDisposed();
             await TaskUtilities.SwitchToBackgroundThread();
+
             try {
-                await CreateBrokerSessionAsync(name, rCommandLineArguments, cancellationToken);
-                var webSocket = await ConnectToBrokerAsync(name, cancellationToken);
+                bool sessionExists = await IsSessionRunningAsync(name);
```

Comment:
This is the one line that will need to change to completely disable the reconnect feature, if we decide to not do it for this milestone. Everything else can remain in.